test/transforms.jl

```diff
@@ -1,6 +1,45 @@
 @testset "Transforms" begin
   # using MersenneTwister for compatibility between Julia versions
   rng = MersenneTwister(42)
+  @testset "Levels" begin
+    a = categorical(["yes","no","no","no","yes"]) 
+    b = categorical([1,2,4,2,8],ordered=false) 
+    c =categorical([1,2,1,2,1]) 
+    d= [1,23,5,7,7]
+    t = Table(; a, b, c, d)
+
+    T = Levels(:a => ["yes","no"], :c => [1,2,4], :d => [1,23,5,7])
+    T_same = Levels("a" => ["yes","no"], "c" => [1,2,4], "d" => [1,23,5,7])
```

Comment:
Please remove T_same as it is the same test, except the constructor is different. If you want to test the constructor, please use a separate block of tests.